src/sentry/static/sentry/app/components/events/interfaces/breadcrumbsV2/time.tsx

```diff
@@ -2,38 +2,86 @@ import React from 'react';
 import styled from '@emotion/styled';
 import moment from 'moment';
 
+import {t} from 'app/locale';
 import {defined} from 'app/utils';
 import Tooltip from 'app/components/tooltip';
 import getDynamicText from 'app/utils/getDynamicText';
 import TextOverflow from 'app/components/textOverflow';
 
-const getBreadcrumbTimeTooltipTitle = (timestamp: string) => {
+const fromOrNow = (time: string, timeToCompareWith: string) => {
+  if (
+    !(
+      moment(time).format('HH:mm:ss') === moment(timeToCompareWith).format('HH:mm:ss') &&
```

Comment:
You parse `time` and `timeToCompareWith` multiple times in this method. Could we do that once instead?